src/main/java/seedu/contax/logic/commands/EditPriorityCommand.java

```diff
@@ -0,0 +1,73 @@
+package seedu.contax.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import seedu.contax.commons.core.Messages;
+import seedu.contax.commons.core.index.Index;
+import seedu.contax.logic.commands.exceptions.CommandException;
+import seedu.contax.model.Model;
+import seedu.contax.model.appointment.Appointment;
+import seedu.contax.model.appointment.Priority;
+
+
+/**
+ * Edits the level of an existing priority in the address book
```

Comment:
This javadoc is clearly copy pasted, please fix.

Reply:
thx will modify